### ISSUE TITLE
Autoplay Progress does not work on Mac FireFox

### ISSUE THREAD

**Issue by tvaught91** [nolimits4web/swiper#7348]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://swiperjs.com/demos#autoplay-progress

### Bug description

Using the most recent version of FF (123.0) on Mac 14.3.1, the autoplay progress dot is filled and does not draw the circle.

### Swiper version

11.0.7

### Platform/Target and Browser Versions

FF (123.0) on Mac 14.3.1

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by tvaught91** [comment]
I believe I've figured out the issue.

The css for the element is currently:

```
.autoplay-progress svg {
stroke-dashoffset: calc(125.6 * (1 - var(--progress)));
}
```

The fix, add px after 125.6:

```
.autoplay-progress svg {
stroke-dashoffset: calc(125.6px * (1 - var(--progress)));
}
```

This was tested on Mac Safari, Chrome and FF - All Latest Versions.

**Comment by nolimits4web** [comment]
t0ggles-create swiper
